packages/app-builder-lib/src/configuration.ts

```diff
@@ -134,6 +135,12 @@ export interface CommonConfiguration {
    */
   readonly nativeRebuilder?: "legacy" | "sequential" | "parallel" | null
 
+  /**
+   * The package manager to use for installing dependencies and node module collections.
+   * @default auto
+   */
+  readonly packageManager?: "auto" | PM | null
```

Comment:
CC @beyondkmp, curious on your thoughts on adding this.

For next major semver, I'd like to reorganize `npm/node colletion` properties nested within a centralized property (maybe named `modules`?)